terraform/ec2/mac/main.tf

```diff
@@ -26,6 +26,24 @@ locals {
   private_key_content = var.ssh_key_name != "" ? var.ssh_key_value : tls_private_key.ssh_key[0].private_key_pem
 }
 
+#####################################################################
+# Prepare Parameters Tests
+#####################################################################
+
+locals {
+  validator_config        = "parameters.yml"
+  final_validator_config  = "final_parameters.yml"
+  cloudwatch_agent_config = "agent_config.json"
+  instance_temp_directory = "/tmp"
+}
+
+resource "local_file" "update-validation-config" {
```

Comment:
This function will replace all the place holder in the generator configuration (e.g value_per_minute to signal the metrics per minute sending to CWA, etc) and [be used in the validator ](https://github.com/aws/amazon-cloudwatch-agent-test/blob/main/validator/models/validation_config.go#L29-L48) later on